README.md

```diff
@@ -186,6 +186,10 @@ Although [kompose](https://github.com/kubernetes/kompose) can already convert do
 
 ## Advanced Usage
 
+### Dependency healthchecks
```

Comment:
Heading should be separated by empty line

Reply:
Ditto to below heading

Reply:
Done